packages/locale/src/format/cldr/CldrZoneTextPrinterParser.js

```diff
@@ -35,11 +46,34 @@ export default class CldrZoneTextPrinterParser {
         requireNonNull(textStyle, 'textStyle');
         requireInstance(textStyle, TextStyle, 'textStyle');
         this._textStyle = textStyle;
-        Cldr.load(cldrData('supplemental/likelySubtags.json'));
-        Cldr.load(cldrData('supplemental/metaZones.json'));
+        loadCldrData('supplemental/likelySubtags.json');
+        loadCldrData('supplemental/metaZones.json');
     }
 
-    _resolveZoneIdText(cldr, zoneId, style, type, mapZones) {
```

Comment:
Further down, this method is stilled called with mapZones, they should probably replaced by calls to `_cachedResolveZoneIdText`